cpp/src/IceStorm/InstrumentationI.cpp

```diff
@@ -24,24 +24,19 @@ namespace
         public:
             Attributes()
             {
-                add("parent", &TopicHelper::getService);
```

Comment:
I removed this "parent" attribute for topic. Previously it was the service name.

Reply:
This seems unrelated to the simplification of the configuration, and has the potential for breaking existing code.

Why not simply keep the service name in IceStorm, including here for instrumentation?
Any IceBox service has a service name. We can't remove the service name from IceBox.